prefetch.go

```diff
@@ -85,12 +85,16 @@ func updateDBDownloadedFile(repoName string, fileName string) {
 			var existentPkg Package
 			prefetchDB.First(&existentPkg, "packages.package_name = ? and packages.arch = ? AND packages.repo_name = ?", pkg.PackageName, pkg.Arch, pkg.RepoName)
 			if existentPkg.PackageName == "" {
-				prefetchDB.Save(&pkg)
+				if db := prefetchDB.Save(&pkg); db.Error != nil {
+					log.Printf("db error: %v\n", db.Error)
```

Comment:
I see a lot of places where the errors are silenced (not returned to the caller). It is ok to keep this way in this CL but in the future it worth to review all these places and make sure errors are not mistakenly silenced. Usually errors returned to the caller and caller handles it.